app/repositories/MovementRepository.scala

```diff
@@ -53,4 +53,9 @@ class MovementRepository @Inject()(mc: ReactiveMongoComponent)(implicit ec: Exec
         case None        => save(movementCache)
       }
 
+  def delete(pid: String): Future[Unit] =
```

Comment:
There's a `removeByPid` method

Reply:
Wasn't when I asked on the channel if we needed on

Reply:
deleted 'delete'